synapse/lib/aha.py

```diff
@@ -1278,8 +1287,7 @@ def getMyUrl(self, user='root'):
         return f'ssl://{host}:{port}?certname={user}@{network}'
 
     async def getAhaClone(self, iden):
-        lkey = s_common.uhex(iden)
-        byts = self.slab.get(lkey, db='aha:clones')
+        byts = self.slab.get(iden.encode(), db='aha:clones')
```

Comment:
I'd like to stick with the `uhex()` value ( and eventually convert the other uses to it as well )

Reply:
yup :) haven't changed them back yet